src/extension.ts

```diff
@@ -55,14 +56,15 @@ export function activate(context: vscode.ExtensionContext) {
 		zooKeeperProvider.refresh()
 	));
 
-	context.subscriptions.push(vscode.commands.registerCommand('visualZooKeeper.configureServer', async () => {
-		let server = await vscode.window.showInputBox({
-			title: 'Configure ZooKeeper Server',
-			placeHolder: 'Comma separated host:port pairs,each represents a ZooKeeper server.'
-		});
-		if (server) {
-			zkClient.createClient(server);
+	context.subscriptions.push(vscode.commands.registerCommand('visualZooKeeper.configureServer', async (server: string) => {
+		if (!server) {
```

Comment:
Will this prevent users from exiting?

Reply:
No, I've been using it and haven't run into any issues. Adding `server` as an argument just allows configuration to happen automatically if a server string is configured (L146). Otherwise it works as it did, with the input box